docs/content/preview/reference/configuration/default-ports.md

```diff
@@ -79,11 +79,14 @@ YugabyteDB servers expose time series performance metrics in the [Prometheus exp
 <target>/prometheus-metrics
 ```
 
-You can access the Prometheus server on port `9090` of the YugabyteDB Anywhere node, and you can see the list of targets at the `http://<yugaware-ip>:9090/targets`. In particular, note port `9300` for node-level metrics:
+You can access the Prometheus server on port 9090 of the YugabyteDB Anywhere node, and you can see the list of targets at `http://<yugaware-ip>:9090/targets`. In particular, note port 9300 for node-level metrics:
 
-| Service      | Port |
-| ------------ | ---- |
-| Node metrics | 9300 |
+| Service           | Port |
+| ----------------- | ---- |
+| Prometheus server for YugabyteDB Anywhere | 9090 |
+| Node metrics      | 9300 |
```

Comment:
This page could use some improvement. Maybe we can take it up along with the network diagram ticket. We need to explicitly call out that YBA -> DB node 9300 needs to be open but that isn't clear from this line.